bpython/curtsiesfrontend/repl.py

```diff
@@ -1903,6 +1948,11 @@ def initialize_interp(self):
         del self.coderunner.interp.locals["_Helper"]
 
     def getstdout(self):
```

Comment:
Heads up that this method is also called by write2file(), copy2clipboard(), and pastebin() in bpython/repl.py so we may or may not want to change its behavior (depends on whether the changes make sense for those methods too)

Reply:
Oh whoops, I accidentally left that outdated TODO in there. Yeah I agree, we shouldn't need to change this method.